src/util.js

```diff
@@ -12,13 +12,58 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(internal) {
+(function(internal, testing) {
 
   function inherit(childCtor, parentCtor) {
     childCtor.prototype = Object.create(parentCtor.prototype);
     childCtor.prototype.constructor = childCtor;
   }
 
+  /*
```

Comment:
I prefer it to be consistent if we're not actually planning to do JSDoc but it's not a big deal. My nits are mainly based on things Doug nitpicked about when I started on Web Animations :P